src/Microsoft.MobileBlazorBindings/Elements/Handlers/ViewHandler.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using XF = Xamarin.Forms;
+
+namespace Microsoft.MobileBlazorBindings.Elements.Handlers
+{
+    public partial class ViewHandler : VisualElementHandler
+    {
+        public override bool ApplyAdditionalAttribute(ulong attributeEventHandlerId, string attributeName, object attributeValue, string attributeEventUpdatesAttributeName)
+        {
+            switch (attributeName)
+            {
+                case nameof(View.GestureRecognizers):
+                    ViewControl.GestureRecognizers.Clear();
+                    if (attributeValue != null)
+                    {
+                        ((AttributeValueHolder)attributeValue)(out object value);
+                        var gestures = ((IList<XF.IGestureRecognizer>)value);
+                        foreach (var gesture in gestures)
```

Comment:
Just a question. Is it better to use `for` loop here instead of `foreach`?

Reply:
I think unless there's some good reason to change it, `foreach` is logically what is needed here. That is, just go through every item in the list. Don't need an index or anything.